cves/kernel/CVE-2014-2309.yml

```diff
@@ -465,13 +484,14 @@ CWE_instructions: |
     CWE: [123, 456]     # also ok
     CWE: 123            # also ok
 CWE:
-- 119
+- 911
 CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+  Manually confirmed. I used improper update to reference count because thats essentially
+  what was happening in the code where advertisment packages were getting counted incorrectly and 
+  not being set to zero which lead to a DNS vulnerability
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: Flooding with garbage
 CVSS:
```

Comment:
I would put the CVSS found on the NVD page here (looks like it only exists for CVSS 2.0)